analyzers/src/SonarAnalyzer.RuleDescriptorGenerator/SonarAnalyzer.RuleDescriptorGenerator.csproj

```diff
@@ -5,6 +5,9 @@
     <RootNamespace>SonarAnalyzer.RuleDescriptorGenerator</RootNamespace>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6</TargetFramework>
+    <!-- https://github.com/SonarSource/sonar-dotnet/issues/6711 -->
+    <!-- Output into a folder shared between Debug and Release, without a TFM subdirectory -->
+    <OutDir>bin</OutDir>
```

Comment:
This may have undesired and hard to detect side effects but from a practical point of view it does the job pretty well.

Reply:
The `dotnet run` worked, but had unpleasant consequences :/ 

I'll try a different approach that will solve this, simplify the pipeline and (re)use principle we already have for binaries.